ArduPlane/Attitude.cpp

```diff
@@ -8,6 +8,9 @@
 float Plane::get_speed_scaler(void)
 {
     float aspeed, speed_scaler;
+    if (plane.speed_scaling_surpressed) { //scaling is surpressed during climb phase of automatic takeoffs with no airspeed sensor being used due to problems with inaccurate airspeed estimates
+        return 1.0f ;
```

Comment:
I think your better to constrain it to a max of 1 or 1.5 at the end, that way we will still lower it with airspeed.

Reply:
the problem is that "airspeed" can be WAY off and already too high on the launch, instead of being too low on the launch....the case I cited where you land, pick up without disarming, and relaunch is the problem..you can start off with "airspeeds" like 14m/s even though you are not moving...so you pay for it during the initial climb by having PIDs surpressed below a scale factor of 1...my thought was that rarely do you overspeed on an autolaunch at the end of the climb...presumably you are climbing at around cruise airspeed or a little less...
all that said, just limiting scaler to <=1 during the climb would probably be okay...it wont oscillate even if the PIDs are a little lower than desired

Reply:
changed to 1 or less....will stop the oscillations ....having too low pids is better than too high